conf/nextclound-conf.conf

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
This file should probably be called nginx-nextcloud.conf to make it more obvious it is a nginx conf and similar in naming convention to the other .conf files.